createdatabase fails when `autocommit` is `false`

when running TPC-C against cockroach, the recent change that sets `autoCommit` to `false` (meaning explicit transactions), doesn't play well with dropping and recreating the schema.  the following error is generated...

```
[INFO ] 2021-10-06 10:13:13,113 [main]  com.oltpbenchmark.DBWorkload main - Creating new TPCC database...
Exception in thread "main" java.lang.RuntimeException: Unexpected error when trying to create the tpcc database
	at com.oltpbenchmark.api.BenchmarkModule.createDatabase(BenchmarkModule.java:259)
	at com.oltpbenchmark.api.BenchmarkModule.createDatabase(BenchmarkModule.java:237)
	at com.oltpbenchmark.DBWorkload.runCreator(DBWorkload.java:593)
	at com.oltpbenchmark.DBWorkload.main(DBWorkload.java:389)
Caused by: org.postgresql.util.PSQLException: ERROR: table "warehouse" is being dropped, try again later
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2565)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2297)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:322)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:481)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:401)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:322)
	at org.postgresql.jdbc.PgStatement.executeCachedSql(PgStatement.java:308)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:284)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:279)
	at com.oltpbenchmark.util.ScriptRunner.runScript(ScriptRunner.java:111)
	at com.oltpbenchmark.util.ScriptRunner.runScript(ScriptRunner.java:67)
	at com.oltpbenchmark.api.BenchmarkModule.createDatabase(BenchmarkModule.java:257)
```
Setting the `autoCommit` parameter back to `true` fixes this problem.  I think this change either needs to be reverted or needs to configurable on a per database basis.  This works...`ScriptRunner runner = new ScriptRunner(conn, true, true);`

--- a/src/main/java/com/oltpbenchmark/api/BenchmarkModule.java
+++ b/src/main/java/com/oltpbenchmark/api/BenchmarkModule.java
@@ -248,7 +248,7 @@ public abstract class BenchmarkModule {
     public final void createDatabase(DatabaseType dbType, Connection conn) {
         try {
             String ddlPath = this.getDatabaseDDLPath(dbType);
-            ScriptRunner runner = new ScriptRunner(conn, false, true);
+            ScriptRunner runner = new ScriptRunner(conn, true, true);
 
             if (LOG.isDebugEnabled()) {
                 LOG.debug("Executing script [{}] for database type [{}]", ddlPath, dbType);

